mne/viz/montage.py

```diff
@@ -39,6 +42,24 @@ def plot_montage(montage, scale_factor=20, show_names=True, kind='topomap',
                         "mne.channels.montage.DigMontage")
     if kind not in ['topomap', '3d']:
         raise ValueError("kind must be 'topomap' or '3d'")
+
+    dists = cdist(montage.pos, montage.pos)
+    # only consider upper triangular part by setting the rest to np.nan
+    dists[np.tril_indices(dists.shape[0])] = np.nan
+    dupes = np.argwhere(np.isclose(dists, 0))
+    if dupes.any():
+        n_chans = montage.pos.shape[0]
+        n_dupes = dupes.shape[0]
+        idx = np.setdiff1d(montage.selection, dupes[:, 1]).tolist()
+        logger.info("{} duplicate electrode labels found: ".format(n_dupes))
+        logger.info(", ".join([ch_names[d[0]] + "/" + ch_names[d[1]]
+                               for d in dupes]))
+        logger.info("Plotting {} unique labels.".format(n_chans - n_dupes))
+        montage.ch_names = [montage.ch_names[i] for i in idx]
+        ch_names = montage.ch_names
+        montage.pos = montage.pos[idx, :]
+        montage.selection = np.arange(n_chans - n_dupes)
+
```

Comment:
Is there some simple way to add a test? Check the number of points in some `matplotlib` object when plotting the `montage` with dups?